soh/soh/Enhancements/randomizer/3drando/spoiler_log.cpp

```diff
@@ -677,8 +677,38 @@ static void WriteAllLocations(int language) {
 
         // Eventually check for other things here like fake name
         if (location->HasScrubsanityPrice() || location->HasShopsanityPrice()) {
-          jsonData["locations"][location->GetName()]["item"] = placedItemName;
-          jsonData["locations"][location->GetName()]["price"] = location->GetPrice();
+            jsonData["locations"][location->GetName()]["item"] = placedItemName;
+            if (location->GetPlacedItemKey() == ICE_TRAP && location->IsCategory(Category::cShop)) {
```

Comment:
Are scrubs considered in the shop category?

Reply:
I don't believe so, those would be in the cDekuScrub category.